style.css

```diff
@@ -110,6 +124,14 @@ background-color: white;
    font-family: 'Montserrat', sans-serif;
    font-weight:bolder;
```

Comment:
Better use `bold` as bolder is a relative property

Reply:
> Better use `bold` as `bolder` is a relative property

Ok, changed it